src/Microsoft.AspNetCore.Mvc.ViewFeatures/Controller.cs

```diff
@@ -216,12 +216,16 @@ public virtual PartialViewResult PartialView(string viewName, object model)
         /// The view component name. Can be a view component
         /// <see cref="ViewComponents.ViewComponentDescriptor.ShortName"/> or
         /// <see cref="ViewComponents.ViewComponentDescriptor.FullName"/>.</param>
-        /// <param name="arguments">The arguments to pass to the view component.</param>
+        /// <param name="arguments">
+        /// An <see cref="object"/> containing arguments to be passed to the invoked view component method.
```

Comment:
Just copied and pasted from the `IViewComponentHelper`'s `InvokeAsync` method. Want me to update it in all locations?